backend/controller/controller.go

```diff
@@ -821,6 +823,18 @@ func (s *Service) AcquireLease(ctx context.Context, stream *connect.BidiStream[f
 }
 
 func (s *Service) Call(ctx context.Context, req *connect.Request[ftlv1.CallRequest]) (*connect.Response[ftlv1.CallResponse], error) {
+	logger := log.FromContext(ctx)
+
+	requestCounter, err := otel.GetMeterProvider().Meter("ftl_call").Int64Counter(
+		"verb_call",
+		metric.WithDescription("Count of FTL verb calls via the controller"),
+		metric.WithUnit("requests"))
```

Comment:
I think `WithUnit` is supposed to use a UCUM based on the docs:
```go
// WithUnit sets the instrument unit.
//
// The unit u should be defined using the appropriate [UCUM](https://ucum.org) case-sensitive code.
func WithUnit(u string) InstrumentOption { return unitOpt(u) }
```

https://ucum.org/ucum#section-Alphabetic-Index-By-Name

Reply:
ahh thanks for catching that, looks like these are all measurement units, so I'll see if we can just omit unit.

Reply:
indeed we can!